cloudinit/version.py

```diff
@@ -4,8 +4,15 @@
 #
 # This file is part of cloud-init. See LICENSE file for license information.
 
-__VERSION__ = "25.3"
-_PACKAGED_VERSION = "@@PACKAGED_VERSION@@"
+try:
+    from cloudinit import meson_versions  # type: ignore
+
+    __VERSION__ = meson_versions.UPSTREAM_VERSION
+    _DOWNSTREAM_VERSION = meson_versions.DOWNSTREAM_VERSION
+except ImportError:
+    __VERSION__ = "@MISSING_MESON_BUILD_ARTIFACT@"
```

Comment:
Should these two be handled separately? Even in cases of a debugging build, I could see utility in knowing which upstream build you're based off of even if you forgot to set the downstream portion.